manifest.json

```diff
@@ -24,5 +23,6 @@
       "run_at": "document_start",
       "all_frames": true
     }
-  ]
+  ],
+  "host_permissions": ["*://*/*"]
```

Comment:
Are you sure we need this?
```
Unlike the webRequest API, blocking or upgrading requests using the declarativeNetRequest API requires no host permissions when used with the declarativeNetRequest permission.
```

Reply:
This is required. Half of the tests fail if this line is removed.